bin/methods/install.js

```diff
@@ -0,0 +1,22 @@
+const cwd = process.cwd()
+const log = require('log-cb')
+const npm = require('npm')
```

Comment:
Why put it inside `./plugins`?  Currently if you use `npm install` it would just put it in a `node_modules` beside it, then you could just do:
```js
"scripts": [
  "cordlr-role",
  "cordlr-help",
  // ...
]
```